### PR TITLE
feat(#minor): added liquity rewards

### PR DESCRIPTION
**Pull request by jaimehgb**
This PR contains 2 main changes:
- Adding the StabilityPool as a Market entity
  - The stability pool is a pool where people can deposit LUSD which gets then converted to ETH as liquidations occur on the protocol. These depositors are rewarded in LQTY. This pool has its own TVL, deposits, withdrawals, and revenue
  - Revenue from this pool was added directly to the protocol before. Now it is added to the stability pool market entity (and to the protocol as a consequence)
  - Deposits and withdrawals from this pool are now tracked (they weren’t before)
  - LUSD price was hardcoded to 1, now it is fetched from the price lib
- Adding LQTY rewards to the stability pool

Comparing TVL, revenue and deposits from this version and the previous, everything mostly matches. Revenue is a bit different because of LUSD pricing changes. Deposit volume has increased because we weren’t tracking deposits on this pool before.

https://okgraph.xyz/?q=QmNuqVWM1nXknfcgPqmY1DR75spEuKwLCFKAmyxTHgV8bg

### CODE REVIEW

**Comment by jaimehgb** [comment]
@pietydespises You QA’d liquity before. Can you take a quick look to the new values and rewards? I’ve already compared most of it to our currently deployed version and everything makes sense to me, but would be good to get a review from a fresh pair of eyes before deploying it to prod.

File: subgraphs/liquity/src/mapping/TroveManager.ts [hunk 1]
```diff
@@ -185,10 +212,99 @@ function liquidateTrove(event: TroveUpdated, trove: _Trove): void {
...
+ *
+ * @param event Liquidation event
+ */
+export function handleLiquidation(event: Liquidation): void {
+  // To accurately calculate the revenue generated by the Stability pool we
+  // compare the amount of LUSD that was burned to the amount of ETH that was
+  // sent to the StabilityPool. The USD difference of both amounts will be the
+  // StabilityPool revenue.
+  if (!event.receipt) {
+    log.info(
```
Comment (dmelotik) [inline subgraphs/liquity/src/mapping/TroveManager.ts:232]: should this be an error as receipt is expected?

**Review by dmelotik — COMMENTED** [review]
Increment versions 😉. It would also be good to add a few things to the readme ab rewards as well

**Review by dmelotik — APPROVED** [review]
sweet another one done!

**Comment by pietydespises** [comment]
responded to @jaimehgb here: https://github.com/messari/subgraphs/issues/1425#issuecomment-1336295573

**Pull request merged**